server/controllers/stock/reports/stock/exit/exitToDepot.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../../../../../lib/db');
+
+/**
+ * @function fetch
+ * @description fetch stock exit for patient
+ */
+function fetch(depotUuid, dateFrom, dateTo, showDetails) {
+  const IS_EXIT = 1;
+  const EXIT_TO_DEPOT_ID = 8;
+  const sql = `
+  SELECT 
+    i.code, i.text, iu.text AS unit_text, BUID(m.document_uuid) AS document_uuid,
+    SUM(m.quantity) as quantity, m.date, m.description,
+    u.display_name AS user_display_name,
+    dm.text AS document_reference, dd.text AS depot_name
+  FROM stock_movement m
+    JOIN lot l ON l.uuid = m.lot_uuid
+    JOIN inventory i ON i.uuid = l.inventory_uuid
+    JOIN inventory_unit iu ON iu.id = i.unit_id 
+    JOIN depot d ON d.uuid = m.depot_uuid
+    JOIN depot dd ON dd.uuid = m.entity_uuid
+    JOIN user u ON u.id = m.user_id
+    JOIN document_map dm ON dm.uuid = m.document_uuid
+  WHERE m.is_exit = ${IS_EXIT} AND m.flux_id = ${EXIT_TO_DEPOT_ID} AND d.uuid = ? 
+    AND (DATE(m.date) BETWEEN DATE(?) AND DATE(?))
+  GROUP BY i.uuid`;
+
+  const details = ', m.uuid, dd.uuid';
```

Comment:
Hmm... this may be picky, but it might be good to call this `grouping` or something like that.  I was expect this to be extra details in the `SELECT ... FROM` part of the query, but it looks like it adds to the `GROUP BY` portion.